.github/workflows/build_push.yml

```
@@ -155,6 +155,7 @@ jobs:
           KISSKH_SUB_API: ${{ secrets.KISSKH_SUB_API }}
           KAISVA: ${{ secrets.KAISVA }}
           TMDB_API: ${{ secrets.TMDB_API }}
+          WYZIE_API: ${{ secrets.WYZIE.API }}
```

actionlint (1.7.12)

[error] receiver of object dereference "api" must be type of object but got "string"

(expression)